android/res/values-es/strings.xml

```diff
@@ -650,4 +650,19 @@
     <string name="file_information">Información del archivo</string>
     <string name="use_btjunkie">Usar BTJunkie</string>
     <string name="add_transfer">Añadir transferencia</string>
+    <string name="filter_ellipsis">"Filtrando..."</string>
+    <string name="include">Incluir</string>
+    <string name="tip_touch_tags_to"><![CDATA[ <b>TIP:</b> Toca las etiquetas aplicadas para]]></string>
```

Comment:
please move this up to the top, all HTML like tags should go together in a section up top, look at the english file.